src/styles/blocks/nav.scss

```
@@ -0,0 +1,32 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    color: #060b35;
+    list-style: none;
+    line-height: 60px;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    color: inherit;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+
+    transition: transform $transition-time, color $transition-time;
```

i don't see `transform` on hover
```suggestion
    transition: color $transition-time;
```